wireapp/wire-ios-mono
# feat: rely on remote search when searching for users [WPB-4549]

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-4549" title="WPB-4549" target="_blank"><img alt="Story" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10815?size=medium" />WPB-4549</a>  [iOS] Rely on remote search when searching for users
  </td></table>
  <br />
 








































----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

Since team events are not sent anymore, the clients don't maintain a list of team members. Hence, searching for team members must rely on remote results.

### Testing

#### Test Coverage (Optional)


#### How to Test

Start a new conversation, create a group or add participants to conversations. Check that the initial search results are correct as well as the results after a search string has been entered.

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
